libs/community/langchain_community/chat_models/azureml_endpoint.py

```diff
@@ -1,35 +1,44 @@
 import json
-from typing import Any, Dict, List, Optional, cast
+from typing import Any, Dict, List, Optional, cast, Iterator, AsyncIterator, Type, Mapping
 
-from langchain_core.callbacks.manager import CallbackManagerForLLMRun
+from langchain_core.callbacks import (
+    AsyncCallbackManagerForLLMRun,
+    CallbackManagerForLLMRun,
+)
 from langchain_core.language_models.chat_models import BaseChatModel
 from langchain_core.messages import (
     AIMessage,
+    AIMessageChunk,
     BaseMessage,
+    BaseMessageChunk,
     ChatMessage,
+    ChatMessageChunk,
     HumanMessage,
+    HumanMessageChunk,
     SystemMessage,
+    SystemMessageChunk,
+    FunctionMessageChunk,
+    ToolMessageChunk,
 )
-from langchain_core.outputs import ChatGeneration, ChatResult
-
+from langchain_core.outputs import ChatGeneration, ChatResult, ChatGenerationChunk
 from langchain_community.llms.azureml_endpoint import (
     AzureMLBaseEndpoint,
     AzureMLEndpointApiType,
     ContentFormatterBase,
 )
 
 
-class LlamaContentFormatter(ContentFormatterBase):
+class OpenAIStyleContentFormatter(ContentFormatterBase):
     """Content formatter for `LLaMA`."""
 
     def __init__(self) -> None:
         raise TypeError(
-            "`LlamaContentFormatter` is deprecated for chat models. Use "
-            "`LlamaChatContentFormatter` instead."
+            "`OpenAIStyleContentFormatter` is deprecated for chat models. Use "
+            "`CustomOpenAIContentFormatter` instead."
         )
 
 
-class LlamaChatContentFormatter(ContentFormatterBase):
+class CustomOpenAIContentFormatter(ContentFormatterBase):
```

Comment:
If the class supports both chat and completions, then the naming is fine. Otherwise `CustomOpenAIChatContentFormatter` should be used. Python importing may be tricky when you have classes with the same names in different namespaces.